### PR TITLE
Some minor additions for testing CHYT

### PR DESCRIPTION
**Pull request by altvod**

### CODE REVIEW

File: lib/dl_sqlalchemy_chyt/dl_sqlalchemy_chyt/table_engine.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+class YtTable(Engine):
+    def __init__(self, **kwargs: Any):
+        self.engine_kwargs = kwargs
+        super().__init__()
+
+    def get_parameters(self) -> str:  # Method should return something stringable
+        if not self.engine_kwargs:
+            return ""
+        return f"{{{','.join([f'{key}={value}' for key, value in sorted(self.engine_kwargs.items())])}}}"
```
Comment (MCPN) [inline lib/dl_sqlalchemy_chyt/dl_sqlalchemy_chyt/table_engine.py:14]: nitpick: please split this in multiple lines, `))])}}}"` physically hurts to read

File: lib/dl_db_testing/dl_db_testing/database/base.py [hunk 2]
```diff
@@ -158,15 +158,22 @@ class DbTableBase:
     def name(self) -> str:
         return self.table.name
 
-    def insert(self, data: Union[dict, List[dict]]) -> None:
+    def insert(self, data: Union[dict, List[dict]], bulk_insert: bool = False, chunk_size: int = 200) -> None:
```
Comment (MCPN) [inline lib/dl_db_testing/dl_db_testing/database/base.py:161]: `bulk_insert=False` implies `chunk_size=1`, so maybe one new argument is enough?

File: lib/dl_db_testing/dl_db_testing/database/base.py [hunk 3]
```diff
@@ -158,15 +158,22 @@ class DbTableBase:
...
         if not self.can_insert:
             raise RuntimeError("Can't insert into table")
         if isinstance(data, dict):
             self.db.execute(self.table.insert(data))
         elif isinstance(data, list):
-            for row in data:
-                self.insert(row)
+            if bulk_insert:
+                rest = data
+                while rest:
```
Comment (MCPN) [inline lib/dl_db_testing/dl_db_testing/database/base.py:170]: `for chunk in itertools.batched(data, chunk_size):`

**Review by MCPN — CHANGES_REQUESTED** [review]

File: lib/dl_db_testing/dl_db_testing/database/base.py (hunk 3 above)
Comment (altvod) [reply lib/dl_db_testing/dl_db_testing/database/base.py:170]: New in version 3.12.
I know about it and have been waiting for it for years, but we can't use it yet :(

File: lib/dl_db_testing/dl_db_testing/database/base.py (hunk 2 above)
Comment (altvod) [reply lib/dl_db_testing/dl_db_testing/database/base.py:161]: well, ok...
I thought about it, but it didn't seem very user-friendly to me

File: lib/dl_db_testing/dl_db_testing/database/base.py (hunk 3 above)
Comment (MCPN) [reply lib/dl_db_testing/dl_db_testing/database/base.py:170]: oh, my bad, i missed that
can you please add a TODO so we won't forget to use it after upgrading to 3.12?

**Comment by MCPN** [comment]
`main / 🐍[pytest][ext_public]lib/dl_connector_bitrix_gds:ext (pull_request_target) Failing` - failing on main

**Review by MCPN — APPROVED** [review]

**Pull request merged**